elastic/kibana
# [Stack Monitoring] Add log-* index pattern support on SM UI

## Summary

This PR adds `logs-*` index pattern support on places in the SM UI where it queries `filebeat-*` modules, in order to also work with integration packages 

closes #120416

## Screenshots
Cluster Overview UI
<img width="1670" alt="image" src="https://user-images.githubusercontent.com/2767137/185454971-6dd2adff-b2c5-4ece-b648-a17bbc5a2f3c.png">

Elasticsearch overview
<img width="1630" alt="image" src="https://user-images.githubusercontent.com/2767137/185455050-784ae36f-bea3-4afe-8328-169a04153a05.png">

Indices view
<img width="1413" alt="image" src="https://user-images.githubusercontent.com/2767137/185455157-924f4a01-75c1-4185-a2db-dc7f35b06c06.png">

### Testing
**Make sure you have [elastic-package](https://github.com/elastic/elastic-package) installed.**

- This change works with a corresponding PR in the Integrations package. Make sure to follow the instructions in https://github.com/elastic/integrations/pull/4033 to setup the required services
- Configure your local kibana to point to elastic-package stack. See [howto](https://github.com/elastic/observability-dev/blob/8c6b377f5844053d2d202219679d94957a28cc00/docs/infra-obs-ui/stack-monitoring_integration-packages.md#connecting-a-local-kibana)
- Connect to local Kibana at http://localhost:5602 and navigate to the Stack Monitoring UI. Logs should show up in the Elasticsearch views
